spec/stepmod/smrl_resource_converter_spec.rb

```diff
@@ -200,7 +201,7 @@
 
         [stem]
         ++++
-        χ ms = V - E + 2F - L l - 2(S - G s) = 0
+        χ _{ms} = V - E + 2F - L_{l} - 2(S - G^{s}) = 0
```

Comment:
`χ_{ms}` instead of `χ _{ms}`? (need to strip space between)

Reply:
Done